docs/apis/metrics.md

```diff
@@ -0,0 +1,441 @@
+   - Applied to all metrics that were scoped to a task.
+- `metrics.scope.tm.operator`
+  - Default: &lt;host&gt;.taskmanager.&lt;tm_id&gt;.&lt;job_name&gt;.&lt;operator_name&gt;.&lt;subtask_index&gt;
+  - Applied to all metrics that were scoped to an operator.
+
+There are no restrictions on the number or order of variables. Variables are case sensitive.
+
+The default scope for operator metrics will result in an identifier akin to `localhost.taskmanager.1234.MyJob.MyOperator.0.MyMetric`
+
+If you also want to include the task name but omit the task manager information you can specify the following format:
+
+`metrics.scope.tm.operator: <host>.<job_name>.<task_name>.<operator_name>.<subtask_index>`
+
+This could create the identifier `localhost.MyJob.MySource_->_MyOperator.MyOperator.0.MyMetric`.
+
+Note that for this format string an identifier clash can occur should the same job be run multiple times concurrently, which can lead to inconsistent metric data.
+As such it is advised to either use format strings that provide a certain degree of uniqueness by including IDs (e.g &lt;job_id&gt;)
+or by assigning unique names to jobs and operators.
+
+### List of all Variables
+
+- JobManager: &lt;host&gt;
+- TaskManager: &lt;host&gt;, &lt;tm_id&gt;
+- Job: &lt;job_id&gt;, &lt;job_name&gt;
+- Task: &lt;task_id&gt;, &lt;task_name&gt;, &lt;task_attempt_id&gt;, &lt;task_attempt_num&gt;, &lt;subtask_index&gt;
+- Operator: &lt;operator_name&gt;, &lt;subtask_index&gt;
+
+## Reporter
+
+Metrics can be exposed to an external system by configuring a reporter in `conf/flink-conf.yaml`.
+
+- `metrics.reporter.class`: The class of the reporter to use.
+  - Example: org.apache.flink.metrics.reporter.JMXReporter
+- `metrics.reporter.arguments`: A list of named parameters that are passed to the reporter.
+  - Example: --host localhost --port 9010
+- `metrics.reporter.interval`: The interval between reports.
+  - Example: 10 SECONDS
+
+You can write your own `Reporter` by implementing the `org.apache.flink.metrics.reporter.MetricReporter` interface.
+If the Reporter should send out reports regularly you have to implement the `Scheduled` interface as well.
+
+By default Flink uses JMX to expose metrics.
+All non-JMXReporters are not part of the distribution. To use them you have to copy the respective fat jar to the `/lib` folder.
+
+The following sections list the supported reporters.
+
+### JMX
+
+The port for JMX can be configured by setting the `metrics.jmx.port` key. This parameter expects either a single port
+or a port range, with the default being 9010-9025. The used port is shown in the relevant job or task manager log.
+
+### Ganglia (org.apache.flink.metrics.ganglia.GangliaReporter)
+Dependency:
+{% highlight xml %}
+<dependency>
+      <groupId>org.apache.flink</groupId>
+      <artifactId>flink-metrics-ganglia</artifactId>
+      <version>{{site.version}}</version>
+</dependency>
+{% endhighlight %}
+
+Parameters:
+
+- `host` - the ganglia server host
```

Comment:
Note that it is not mandatory to set the bind property. If it is not set then generally host should be set to localhost.

But yes, if it set it to something but localhost then naturally you have to set it to that value, will include that in the docs.